subsystems/art/EnemyVFX.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+from pygame.locals import *
+pygame.init()
+class EnemyVFX:
+    def __init__(self):
+        self.enemyvisuals = []
+        self.enemyNames = []
+        self.deathImages = []
+    def draw(self, surf,enemyName, frameCount, x, y):
+        ind = self.getEnemyIndex(enemyName)
```

Comment:
make sure to handle -1 errors, or at least print or throw an exception with a description of what happened
helps debugging

Reply:
also python will loop -1 to end of loop, which doesn't make sense in this context